classes/Utils/Alerts.php

```diff
@@ -22,6 +22,8 @@ public static function init() {
 		add_filter( 'pum_alert_list', array( __CLASS__, 'integration_alerts' ), 5 );
 		add_filter( 'pum_alert_list', array( __CLASS__, 'translation_request' ), 10 );
 		add_action( 'admin_menu', array( __CLASS__, 'append_alert_count' ), 999 );
+
+		self::php_handler();
```

Comment:
This should probably be hooked into admin_init or similar rather than called directly. IDK are there downsides one way or another? If not I'd go ahead and hook it.

Reply:
Ah! Yes, that should be in a hook. I had put it there while testing and meant to circle back and find an appropriate hook for it. I Will update before merging.